server/package.json

```diff
@@ -123,42 +124,9 @@
     "ts-node": "^10.9.1",
```

Comment:
probably don't need this anymore

Reply:
also `@types/jest` above

Reply:
It is still used in e2e, but I may try to migrate those here in this PR as well